drivers/spi/Kconfig.renesas_ra

```diff
@@ -28,8 +28,8 @@ config SPI_RA_DTC
 config SPI_USE_HW_SS
 	bool "RA MCU SPI Hardware Slave Select support"
 	default y
-	depends on !SOC_SERIES_RA2A1
+	depends on !(SOC_SERIES_RA2A1 || SOC_SERIES_RA4M1)
 	help
-	  Use Slave Select pin instead of software Slave Select.
+	  Use Hardware Slave Select (SSL) instead of Software Slave Select.
```

Comment:
update as per https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-2-inclusive-language

Reply:
updated, thank you